instance/models/instance.py

```diff
@@ -909,6 +465,7 @@ def provision(self):
             log, exit_code = self.deploy()
             if exit_code != 0:
                 server.update_status(provisioning=True, failed=True)
+                self.provision_failed_email(self.ProvisionMessages.PROVISION_ERROR)
```

Comment:
It would be nice to include the ansible log in the email (at least the last part with the failure), or a link to the log in the instance manager console.